database/migrations/2019_03_18_180943_create_module_progresses_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateModuleProgressesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('module_progresses', function (Blueprint $table) {
+            $table->string('user_id');
+            $table->string('current_module');
+            $table->integer('current_page');
+            $table->integer('max_page');
+            $table->timestamps();
```

Comment:
I know Laravel changed their built-in timestamps so where the created_at field was null might want to verify if that is still the case otherwise change it to the suggested snippet

```suggestion
            $table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'));
			$table->timestamp('updated_at')->nullable();		
```